docs/src/prototypes/customToolbar/darkThemeOverrides.ts

```diff
@@ -0,0 +1,227 @@
+import {
+  ComponentStyleFunctionParam,
+  ThemeInput,
+  pxToRem,
+  ToolbarProps,
+  ToolbarItemProps,
+  ToolbarCustomItemProps,
+} from '@stardust-ui/react'
+
+export type CustomToolbarVariables = {
+  isCt?: boolean
+
+  isCtItemDanger?: boolean
+  isCtItemPrimary?: boolean
+  isCtItemIconNoFill?: boolean
+  isCtItemIndicator?: boolean
+  isCtItemWithNotification?: boolean
+
+  ctBorderRadius: string
+  ctBorderStyle: string
+  ctBorderWidth: string
+  ctHeight: string
+
+  ctItemBackground: string
+  ctItemBackgroundHover: string
+  ctItemBorderColorFocus: string
+  ctItemColor: string
+  ctItemColorFocus: string
+  ctItemColorHover: string
+
+  ctItemActiveColor: string
+  ctItemActiveBackground: string
+  ctItemActiveBackgroundOverlay: string
+
+  ctItemDangerBackground: string
+  ctItemDangerColorHover: string
+  ctItemDangerBackgroundHover: string
+
+  ctItemIndicatorPadding: string
+
+  ctItemNotificationBackgroundColor: string
+  ctItemNotificationSize: string
+
+  ctItemPrimaryBackground: string
+  ctItemPrimaryBackgroundHover: string
+  ctItemPrimaryColorHover: string
+}
+
+export const darkThemeOverrides: ThemeInput = {
+  componentVariables: {
+    Toolbar: (siteVars): CustomToolbarVariables => ({
+      ctBorderRadius: '4px',
+      ctBorderStyle: 'solid',
+      ctBorderWidth: '2px',
+      ctHeight: '4rem',
+
+      ctItemBackground: siteVars.colorScheme.default.background1,
+      ctItemBackgroundHover: siteVars.colorScheme.brand.backgroundHover1,
+      ctItemBorderColorFocus: siteVars.colorScheme.default.borderFocus,
+      ctItemColor: siteVars.colorScheme.default.foreground,
+      ctItemColorFocus: siteVars.colorScheme.default.foregroundFocus,
+      ctItemColorHover: siteVars.colorScheme.default.foregroundHover,
+
+      ctItemActiveBackground: siteVars.colorScheme.brand.backgroundHover1,
+      ctItemActiveBackgroundOverlay:
+        'linear-gradient(90deg,rgba(60,62,93,.6),rgba(60,62,93,0) 33%),linear-gradient(135deg,rgba(60,62,93,.6) 33%,rgba(60,62,93,0) 70%),linear-gradient(180deg,rgba(60,62,93,.6) 70%,rgba(60,62,93,0) 94%),linear-gradient(225deg,rgba(60,62,93,.6) 33%,rgba(60,62,93,0) 73%),linear-gradient(270deg,rgba(60,62,93,.6),rgba(60,62,93,0) 33%),linear-gradient(0deg,rgba(98,100,167,.75) 6%,rgba(98,100,167,0) 70%)',
```

Comment:
Let's check before merge whether these colors exists in the color palette.

Reply:
Our plan is not to match redlines (for now), comment added.